test/functional/test_framework/authproxy.py

```diff
@@ -139,17 +139,24 @@ def _request(self, method, path, postdata):
             return self._get_response()
 
     def __call__(self, *args, **argsn):
-        AuthServiceProxy.__id_count += 1
+        # check for a possible wallet endpoint
+        urlAdd = ""
+        method = self._service_name
+        elements = self._service_name.split(".")
+        if len(elements) > 1:
+            urlAdd = "/wallet/" + '.'.join(elements[:-1])
```

Comment:
I think this actually should use `__truediv__` not `__idiv__`,
 which is for python 2.x (https://docs.python.org/3/reference/datamodel.html?highlight=truediv#object.__truediv__)